README.md

```diff
@@ -2617,6 +2617,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Tushar Singh](https://github.com/SINGHxTUSHAR)
   - [Twinkle](https://github.com/twinkletrivaan)
   - [Tyler Snelgrove](https://github.com/tylersnel)
+  - [Theresa Seyram Agbenyegah](https://github.com/Stancy-ai)
```

Comment:
Please add your name at the right position to keep this list in the correct alphabetical order. (It should be line 2592.)

Reply:
oh ok will fix it

Reply:
Change request fixed